src/components/header/header.tsx

```diff
@@ -0,0 +1,91 @@
+import { Component, h, Prop, Event, EventEmitter } from '@stencil/core';
+import { Action } from '../collapsible-section/action';
```

Comment:
If we need to use this component in the global table and object card. We don't need to import the Action from `collabsible-section`. This is the interface we need 
``` 
export interface Action {
    /**
     * The command to send to the commandbus when the action is activated
     */
    command: object;

    /**
     * Icon of the action
     */
    icon?: string;

    /**
     * Label to display next to the action
     */
    label?: string;

    /**
     * True if the action is disabled
     */
    disabled?: boolean;
}